.github/workflows/transifex.yml

```
@@ -1,20 +1,21 @@
-name: Pull and push strings via Transifex
+name: Sync strings via Transifex
```

Nice change. But usually there's something to synchronize data _with_.
```suggestion
name: Sync strings with Transifex
```
I don't know if that really matters.

---

Yeah, I forgot that you said to use "with" after I pushed the change, but I'm still being undecided.

Now you mentioned it, I would change it to it.